### PR TITLE
Docs for most viewed datasets

### PR DESCRIPTION
**Pull request by pablopareja**

### CODE REVIEW

**Review by tiagojsag — COMMENTED** [review]
Thanks for contributing, @pablopareja 

I'll take advantage of your documentation generosity, and ask for two improvements:
- the endpoint accepts an optional `application` or `app` query param, but I can't tell from the code if it actually uses it for anything. Could you test, please?
- There's an optional `limit` query param to limit how many datasets are returned (there's no pagination, just a limit)

https://github.com/resource-watch/graph-client/blob/52176d55884077fe1852fed7c5696e8cf5b09578/app/src/routes/api/v1/graph.router.js#L323

**Comment by pablopareja** [comment]
Thanks for your suggestions for improvement Tiago 👍 
I've been digging in the code and, for some reason, the query used to gather the most viewed datasets doesn't seem to be using the application parameter `QUERY_MOST_VIEWED` ➡️ https://github.com/resource-watch/graph-client/blob/e86d54493809cd8ecb8c930d260bd58bf6b4b7fb/app/src/services/neo4j.service.js#L180
However, in the case of the `queryMostViewedByUser` ➡️  https://github.com/resource-watch/graph-client/blob/e86d54493809cd8ecb8c930d260bd58bf6b4b7fb/app/src/services/neo4j.service.js#L180, the query does use the application parameter. `QUERY_MOST_VIEWED_BY_USER` ➡️  https://github.com/resource-watch/graph-client/blob/e86d54493809cd8ecb8c930d260bd58bf6b4b7fb/app/src/services/neo4j.service.js#L186 

I'll add the docs for the `limit` option anyway.

**Comment by pablopareja** [comment]
@tiagojsag `limit` docs added 😉

File: source/includes/api/_graph.md [hunk 1]
```diff
@@ -524,217 +524,50 @@ curl -X GET https://api.resourcewatch.org/v1/graph/query/most-liked-datasets
...
-			"count": {
-				"low": 1,
-				"high": 0
-			}
 		}
 	]
 }
 ```
+
+## Most viewed datasets
```
Comment (tiagojsag) [inline source/includes/api/_graph.md]: tip for the future: identify the action. 

In this example, "most viewed datasets" doesn't have an action. Are you documenting how to GET it? How to update (PATCH/PUT/POST) it? How to DELETE it?

File: source/includes/api/_graph.md [hunk 2]
```diff
@@ -524,217 +524,50 @@ curl -X GET https://api.resourcewatch.org/v1/graph/query/most-liked-datasets
 				"low": 2,
 				"high": 0
 			}
-		},
```
Comment (tiagojsag) [inline source/includes/api/_graph.md:527]: <3

**Pull request merged**